### PR TITLE
OSDOCS-1455: Updating backup and restore procedures

### PR DESCRIPTION
**Pull request by bergerhoffer**
https://issues.redhat.com/browse/OSDOCS-1455

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@geliu2016 Have you had any luck in testing this again? See my response here to the issue you said that you were having in testing this earlier: https://github.com/openshift/openshift-docs/pull/22300#discussion_r527363368

(Note that these update moved out of that PR and into this new one)

I think that the problem may have been because your cluster had "etcd" in the name, so if you could try with a cluster without that to see if this succeeds or not. And then if that was the problem, we can look to update the instructions to not have an issue with a cluster with "etcd" in the name.

File: day_two_guide/topics/proc_restoring-master.adoc [hunk 1]
```diff
@@ -68,13 +68,14 @@ $ diff /tmp/current_packages.txt ${MYBACKUPDIR}/packages.txt
 package lists.
 
 . Restore a system certificate by copying the certificate to the
-`/etc/pki/ca-trust/source/anchors/` directory and execute the `update-ca-trust`:
+`/etc/pki/ca-trust/source/anchors/` directory and execute `update-ca-trust`:
 +
 ----
 $ MYBACKUPDIR=*/backup/$(hostname)/$(date +%Y%m%d)*
-$ sudo cp ${MYBACKUPDIR}/external_certificates/my_company.crt /etc/pki/ca-trust/source/anchors/
+$ sudo cp ${MYBACKUPDIR}/etc/pki/ca-trust/source/anchors/<certificate> /etc/pki/ca-trust/source/anchors/ <1>
```
Comment (bergerhoffer) [inline day_two_guide/topics/proc_restoring-master.adoc:75]: @xingxingxia Can you please review this update? It is to remove the reference to "my_company.crt", which you pointed out was not referenced in the backup document. Thanks!

File: day_two_guide/topics/proc_restoring-node.adoc [hunk 2]
```diff
@@ -65,13 +65,15 @@ $ diff /tmp/current_packages.txt ${MYBACKUPDIR}/packages.txt
 package lists.
 
 . Restore a system certificate by copying the certificate to the
-`/etc/pki/ca-trust/source/anchors/` directory and execute the `update-ca-trust`:
+`/etc/pki/ca-trust/source/anchors/` directory and execute `update-ca-trust`:
 +
 ----
 $ MYBACKUPDIR=*/backup/$(hostname)/$(date +%Y%m%d)*
-$ sudo cp ${MYBACKUPDIR}/etc/pki/ca-trust/source/anchors/my_company.crt /etc/pki/ca-trust/source/anchors/
+$ sudo cp ${MYBACKUPDIR}/etc/pki/ca-trust/source/anchors/<certificate> /etc/pki/ca-trust/source/anchors/
```
Comment (bergerhoffer) [inline day_two_guide/topics/proc_restoring-node.adoc:72]: @xingxingxia The same update is here if you can review as well. Thanks!

**Comment by geliu2016** [comment]
@bergerhoffer I retried it with v3.11.321 and backup/restore both work well, there is minor err about env var ETCD_POD, and I add comments in #22300 (comment), thanks

File: day_two_guide/topics/proc_restoring-master.adoc (hunk 1 above)
Comment (xingxingxia) [reply day_two_guide/topics/proc_restoring-master.adoc:75]: @bergerhoffer this place and below place lgtm, thank you.

Comment (bergerhoffer) [reply day_two_guide/topics/proc_restoring-master.adoc:75]: Thank you!

**Comment by bergerhoffer** [comment]
@geliu2016 We updated that command to be:

```
$ export ETCD_POD=$(oc get pods -n kube-system | grep -o -m 1 '^master-etcd\S*')
```

Can you please retest using this command and see if that works properly for you? And if so, whether this PR looks good now or if you have any other feedback? Thanks!

**Comment by geliu2016** [comment]
@bergerhoffer Yes, it works well in my current env, but I have concern about if it still not exact in some other customer's cluster env, because I'm not familiar with the etcd pods name rules. Is it possible reminder customer find etcd pods name in project: kube-system by themselves? or we may confirm with Sam that "master-etcd" is unique keyword in etcd pods name rules, anyway, it's minor issue, thanks. 

$ oc get pods
NAME                                           READY     STATUS    RESTARTS   AGE
master-api-geliu311master-etcd-nfs-1           1/1       Running   9          1d
master-controllers-geliu311master-etcd-nfs-1   1/1       Running   3          1d
master-etcd-geliu311master-etcd-nfs-1          1/1       Running   0          1d
[root@geliu311master-etcd-nfs-1 ~]$ export ETCD_POD=$(oc get pods -n kube-system | grep -o -m 1 '^master-etcd\S*')
[root@geliu311master-etcd-nfs-1 ~]$ echo $ETCD_POD
master-etcd-geliu311master-etcd-nfs-1

**Comment by bergerhoffer** [comment]
Confirmed above with @hexfusion, merging.

**Pull request merged**